src/containers/MovieContainer.js

```diff
@@ -1,12 +1,30 @@
 import React from 'react';
 import { connect } from 'react-redux';
+import MovieCard from '../components/MovieCard';
 
-const MovieContainer = () => {
-  return (
-    <div>
-      <h1>MovieContainer</h1>
-    </div>
-  );
+const MovieContainer = props => {
+  const { allMovies } = props;
+
+  const displayMovies = allMovies.map(movie => {
+    return (
+      <MovieCard
```

Comment:
This ties to my other question about passing props rather than accessing from the store.  I can't wrap my head around how we would render these cards *without* passing this info as props.